.github/workflows/yaml.yml

```diff
@@ -21,4 +23,4 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Run yamllint
-        run: find . -path \*/vendor -prune -false -o -name \*.y*ml | xargs yamllint
+        run: git ls-files --cached -z -- '*.y*ml' | xargs --null -- yamllint
```

Comment:
Explanation:
- `-z` quotes filenames with special characters and separates each path with NUL byte (that's why when you call this in CLI you get one long string, instead of multiple lines)
- `xargs --null` then handles this NUL byte and applies YAML linter to each file

I am not 100% sure if it's needed here, we could use `git ls-files` without `-z` and then `xargs` would work on new lines. But both ways work, so 👍.

Reply:
I believe we don't have files with spaces in name, but yeah, your approach is better here:

<img width="1214" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/1f2a9988-967c-4316-bca9-9da150afc452">